src/bundle/Resources/config/services/dashboard.yaml

```diff
@@ -44,6 +44,10 @@ services:
         autowire: true
         public: false
 
+    Ibexa\AdminUi\Tab\Dashboard\PagerLocationToDataMapper:
```

Comment:
I don't remember exactly but there is a high chance that `PagerContentToDataMapper` is only used here in tab. If so we can deprecate it. But that needs to be checked product-wise.

Reply:
Yes, it's used only in Dashboard tabs, I've deprecated it.